### PR TITLE
Feat/animator

### PR DESCRIPTION
**Pull request by luzhuang**
feat: add play backward doc, demo and opt animator doc

### CODE REVIEW

**Review by eyworldwide — APPROVED** [review]

**Review by GuoLei1990 — APPROVED** [review]
+1

File: playground/skeleton-animation-additive.ts [hunk 1]
```diff
@@ -54,22 +56,53 @@ engine.resourceManager
...
+
     if (animations) {
       animations.forEach((clip: AnimationClip) => {
-        if (clip.name === "agree") {
-          const animatorState = animatorStateMachine.addState(clip.name);
-          animatorState.clip = clip;
-        }
-        if (clip.name === "sneak_pose") {
+        if (clip.name.includes("pose")) {
+          console.log(clip.name)
```
Comment (cptbtptpbcptdtptp) [inline playground/skeleton-animation-additive.ts:66]: Delete this line.

File: playground/skeleton-animation-additive.ts [hunk 2]
```diff
@@ -54,22 +56,53 @@ engine.resourceManager
...
           const animatorState2 = animatorStateMachine1.addState(clip.name);
           animatorState2.clip = clip;
           animatorState2.clipStartTime = 1;
+          animationNames2.push(clip.name);
+        } else {
+          const animatorState = animatorStateMachine.addState(clip.name);
+          animatorState.clip = clip;
+          animationNames.push(clip.name);
+          console.log(clip.name)
```
Comment (cptbtptpbcptdtptp) [inline playground/skeleton-animation-additive.ts:75]: Delete this line.

File: docs/animator.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ The [Animator](${api}core/Animator) can organize [AnimationClips](${api}core/Ani
 <playground src="skeleton-animation.ts"></playground>
 
 ## Basic usage
-After loading the glTF model, the engine will automatically add an Animator component to the model's root entity , and add the animation clips in the model to it. You can get the Animator component directly on the root entity of the model and play the specified animation.
+After loading the glTF model, the engine will automatically add an Animator component to the model's root entity , and add the animation clips in the model to it. You can get the Animator component directly on the root entity of the model and play the specified animation. When the value is negative
```
Comment (cptbtptpbcptdtptp) [inline docs/animator.md:12]: "When the value is negative"

**Pull request merged**